addons/website/tests/asset_tag.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <asset id="theme_default.test_asset_tag_aaa" name="Test asset tag (init active => keep active => update active)">
```

Comment:
we don't create tests/fake data in real modules, this should be move to `test_website` which only purpose is exactly to do this (see manifest)

Reply:
@rdeodoo Should I also move the existing `website/tests/template_qweb_test.xml` to `test_website` ? (maybe in a distinct PR)
Note that those files are not loaded in a normal module load - they are not part of `website`'s `data` manifest entry. They are only used during tests that explicitly trigger their import.

Reply:
You can indeed, but in master. This one is in `website` because I introduced `test_website` [introduced](https://github.com/odoo/odoo/commit/ef03db9edd9472201cb2c08a32d20ff0f33a5fdf) after that [file](https://github.com/odoo/odoo/commit/9cd982bcc811cacb42f5c08db139043d2734b891).

> Note that those files are not loaded in a normal module load - they are not part of website's data manifest entry. They are only used during tests that explicitly trigger their import.

Still, it's noising the file hierarchy for no good reason (it's really not that big of a deal but since we have a module just for that..)

Reply:
Moved to https://github.com/odoo/odoo/pull/109349